models/DataObject/ClassDefinition.php

```diff
@@ -179,6 +179,11 @@ class ClassDefinition extends Model\AbstractModel
         ]
     ];
 
+    /**
+     * @var bool
+     */
+    public $enableGridLocking = true;
```

Comment:
I would keep this default `false` just like other options :)

Reply:
I've changed it to false.